### DISCUSSION TITLE
Setting default LaF for NBP app?

### DISCUSSION THREAD

**Discussion by sjaenick** [apache/netbeans#5047] [Q&A] [answered]
Hi there,

recent NB IDE releases (at least on Linux) have switched to FlatLaf as the default, while my own NBP application, which
is currently based on RELEASE160, still comes up with the Metal LaF. Obviously, I can switch to FlatLaF manually, but I'd
like to use it as the default. Any hints how to achieve this?

Thanks in advance.

**Comment by neilcsmith-net** [comment, marked as answer]
The same way that the IDE does this - branding. See https://github.com/apache/netbeans/pull/3332

FlatLaf is the default IDE for the IDE across all OS, but the old defaults were kept for the platform for backwards compatibility / consistency.
